1d-life/src/1d-life.js

```diff
@@ -40,8 +40,29 @@
    */
   function getNewVal(lifeState, x) {
     // !!! IMPLEMENT ME
+    const oneDArray = lifeState;
 
-    return 0; // instead of this
```

Comment:
nice job Alif, looks like you can simplify this a little but you got the rules down.